docs/src/content/docs/extract/bank-workflows/adb.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Asian Development Bank (ADB)
+description: A guide in my new Starlight docs site.
+---
+
+## Insert Projects Workflow
+
+The following highlighted fields are webscraped:
```

Comment:
Not directly relevant to this PR, but this seems like a good format for this documentation. I think having a couple filled out examples would be useful, but I'm guessing the rest would make sense to wait if:
- this stuff is easier to fill out while you are working on it
- you haven't worked directly on any of these in awhile

Then it might make sense to just wait to fill each of the rest out until you need to.